corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -101,6 +101,10 @@ def log_error(self, message):
         _logger.error(message)
         print('[ERROR] {}'.format(message))
 
+    def log_warning(self, message):
+        _logger.warning(message)
+        print('[WARNING] {}'.format(message))
+
```

Comment:
These log methods might be getting to the point where you'd want to DRY them by getting them to call a generic `_log()` method like
```python
    def _log(self, level, message):
        _logger.log(level, message)
        if level != logging.DEBUG or self.debug:
            print('[{}] {}'.format(logging.getLevelName(level), message))
```
with, say, 
```python
    def log_debug(self, message):
        self._log(logging.DEBUG, message)
```
(the way [logging](https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L1433) itself does).

Reply:
This is a good suggestion. I do it in a future PR.